### PR TITLE
Set role and aria-label for charts

### PR DESCRIPTION
**Pull request by krkshitij**

### CODE REVIEW

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts [hunk 1]
```diff
@@ -48,17 +51,25 @@ export class HorizontalBarChart extends FASTElement {
...
+    this.elementInternals.role = 'region';
+  }
+
   connectedCallback() {
     super.connectedCallback();
 
     validateChartPropsArray(this.data, 'data');
 
     this._isRTL = getRTL(this);
+    this.elementInternals.ariaLabel = this.chartTitle || `Horizontal bar chart with ${this.data.length} bars.`;
```
Comment (AtishayMsft) [inline packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:72]: the data is a 2d object, first representing each row and second shows bars in each row. Which property are you using

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts [hunk 2]
```diff
@@ -288,7 +299,7 @@ export class HorizontalBarChart extends FASTElement {
       .append('div')
       .append('span')
       .attr('class', 'chart-title')
-      .text(data?.chartTitle ? data?.chartTitle : '');
+      .text(data?.barTitle ? data?.barTitle : '');
```
Comment (AtishayMsft) [inline packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:302]: Is the title for each row or all rows together.

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts [hunk 3]
```diff
@@ -317,7 +328,7 @@ export class HorizontalBarChart extends FASTElement {
       .attr('height', 12)
       .attr('width', 100 + '%')
       .attr('class', 'svg-chart')
-      .attr('aria-label', data?.chartTitle ? data?.chartTitle : '')
+      .attr('aria-label', data?.barTitle ? data?.barTitle : '')
```
Comment (AtishayMsft) [inline packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:331]: lets use a default aria label

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 2 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:302]: title for each row

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 3 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:331]: title for each row should always be present. should I set it regardless?

Comment (AtishayMsft) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:331]: very corner case. But lets still set it.

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 1 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:72]: the first one. it basically announces the total number of bars (or rows) present.

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 2 above)
Comment (AtishayMsft) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:302]: lets call it chartSeriesTitle

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 1 above)
Comment (AtishayMsft) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts:72]: Then call it - Horizontal bar chart with ${this.data.length} categories.

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 3 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts]: done

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 2 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts]: done

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts (hunk 1 above)
Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts]: done

File: packages/charts/chart-web-components/src/donut-chart/donut-chart.ts [hunk 4]
```diff
@@ -77,6 +84,7 @@ export class DonutChart extends FASTElement {
 
     this.legends = this.getLegends();
     this._isRTL = getRTL(this);
+    this.elementInternals.ariaLabel = this.data.chartTitle || `Donut chart with ${this.data.chartData.length} slices.`;
```
[the remark this answers is not in the document]
Comment (krkshitij) [reply packages/charts/chart-web-components/src/donut-chart/donut-chart.ts]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**